cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -182,7 +182,7 @@ public void testFallbackToAvgAvailable() throws NotEnoughValidWindowsException {
         metricSampleAggregator.aggregate(clusterAndGeneration(metadata.fetch()),
                                          NUM_WINDOWS * WINDOW_MS,
                                          new OperationProgress());
-    assertTrue(result.valuesAndExtrapolations().isEmpty());
+    assertEquals(NUM_WINDOWS - 2, result.valuesAndExtrapolations().get(PE).windows().size(), 0);
```

Comment:
Would it make the code more clear if we add a comment to clarify why do we expect these values?

Reply:
Add a comment. The detailed explanation is same as above.